### ISSUE TITLE
Transcode queue hung because torrent was removed on site

### ISSUE THREAD

**Issue by talentedmrripley** [karamanolev/WhatManager2#81]
A torrent that I added to be transcoded took a long time to download, very few and slow seeders.  In between the time that it was added and completed it was removed on the site.  On the transcode page it showed as torrent removed.  Items that I added after it completed downloading but never began the transcode/upload process.  After troubleshooting why my transcodes weren't going I deleted the removed torrent from the DB and immediately the other queue items began to be transcoded.

**Comment by karamanolev** [comment]
Yes, that's indeed an issue. I didn't get around to fixing it when I spent most of my time working on WM2. PRs that fix this are welcome.
